lib/assets/chart_cell/main.js

```diff
@@ -150,7 +150,7 @@ export function init(ctx, payload) {
         fields: payload.fields,
         dataOptions: payload.data_options,
         missingDep: payload.missing_dep,
-        chartOptions: ["bar", "line", "point", "area"],
+        chartOptions: ["bar", "line", "point", "area", "boxplot"],
```

Comment:
While we are at it, let's make point the default type as it has a better chance of showing something reasonable for arbitrary data, wdyt?

```suggestion
        chartOptions: ["point", "bar", "line", "area", "boxplot"],
```

Reply:
Sure! Looks good :)